libsql-server/src/namespace/meta_store.rs

```diff
@@ -0,0 +1,397 @@
+#![allow(dead_code)]
+
+use std::{collections::HashMap, sync::Arc};
+
+use crate::{
+    auth::{Authenticated, Authorized, Permission},
+    connection::{
+        config::DatabaseConfig,
+        program::{Program, Step},
+    },
+    database::Database,
+    error::Error,
+    query::{Params, Query, Value},
+    query_analysis::{Statement, StmtKind},
+    query_result_builder::{IgnoreResult, QueryResultBuilder},
+    Result,
+};
+use parking_lot::Mutex;
+use tokio::sync::{
+    mpsc,
+    watch::{self, Receiver, Sender},
+};
+
+use super::{MakeNamespace, Namespace, NamespaceName};
+
+use crate::connection::Connection;
+
+static INTERNAL_NAMESPACE: &str = "_libsql_server_internal";
```

Comment:
This is wrong. If used as dbId it will mean all sqld instances in all clusters will share the same bottomless backup for their internal namespaces

Reply:
This is going away with the new version of this but when I talked to athos before I implemented this I was told that each cluster gets its own folder and thus can have its own internal namespace. But anyways this is going away now.